tests/aggregation/test_binary_relevance_aggregation.py

```diff
@@ -0,0 +1,60 @@
+"""
+Simple aggregation tests.
+"""
+import pandas as pd
+import pytest
+from pandas.testing import assert_series_equal
+import typing as tp
+
+from crowdkit.aggregation import BinaryRelevance, MajorityVote, DawidSkene, Wawa, GLAD
+
+
+@pytest.fixture
+def data_toy_binary_relevance() -> pd.DataFrame:
+    return pd.DataFrame(
+        [
+            ['t1', 'w1', ['house', 'tree']],
+            ['t1', 'w2', ['house']],
+            ['t1', 'w3', ['house', 'tree', 'grass']],
+            ['t2', 'w1', ['car']],
+            ['t2', 'w2', ['car', 'human']],
+            ['t2', 'w3', ['train']],
+        ],
+        columns=['task', 'worker', 'label']
+    )
+
+
+@pytest.fixture
+def binary_relevance_toy_result() -> pd.Series:
+    result = pd.Series([['house', 'tree'], ['car']], index=['t1', 't2'], name='agg_label')
+    result.index.name = 'task'
+    return result
+
+
+@pytest.mark.parametrize(
+    'aggregator, args', [(MajorityVote, {}), (DawidSkene, {'n_iter': 10}), (Wawa, {}), (GLAD, {})]
```

Comment:
See my comment on sklearn above. Here we pass the classes, but sklearn passes the instances. It might be confusing to those who are used to sklearn.